plugin-debezium-mongodb/src/main/java/io/kestra/plugin/debezium/mongodb/Trigger.java

```diff
@@ -133,6 +133,16 @@ public class Trigger extends AbstractTrigger implements PollingTriggerInterface,
     @Builder.Default
     protected Property<String> stateName = Property.ofValue("debezium-state");
 
+    @Schema(
+        title = "How to commit the offsets to the KV Store.",
+        description = """
+            Possible values are:
+            - ON_EACH_BATCH: after each batch of records consumed by this trigger, the offsets will be stored in the KV Store. This avoids any duplicated records being consumed but can be costly if many events are produced.
+            - ON_STOP: when this trigger is stopped or killed, the offsets will be stored in the KV Store. This avoid any un-necessary writes to the KV Store, but if the trigger is not stopped gracefully, the KV Store value may not be updated leading to duplicated records consumption."""
+    )
+    @Builder.Default
+    protected Property<io.kestra.plugin.debezium.AbstractDebeziumRealtimeTrigger.OffsetCommitMode> offsetsCommitMode = Property.ofValue(io.kestra.plugin.debezium.AbstractDebeziumRealtimeTrigger.OffsetCommitMode.ON_EACH_BATCH);
```

Comment:
Import the classes and remove the packages like this everywhere else too:

```suggestion
    protected Property<OffsetCommitMode> offsetsCommitMode = Property.ofValue(OffsetCommitMode.ON_EACH_BATCH);
```